packages/core/src/messages/packet/header/createHeader.ts

```diff
@@ -181,7 +181,7 @@ export async function createHeader(header: Header, peerIds: PeerId[]) {
           )
         }
 
-        u8aXOR(true, header.beta, PRG.createPRG(key, iv).digest(0, BETA_LENGTH))
+        u8aXOR(true, header.beta, PRG.createPRG(params).digest(0, BETA_LENGTH))
```

Comment:
why are we still XORing in core, rather than using a function from crypto?

Reply:
to be taggled in next PR